.teamcity/generated/ImageValidation.kts

```diff
@@ -17,86 +17,79 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.finishBuildTrigger
 
 object image_validation: BuildType({
 
-		name = "Validation of Size Regression - Docker Image (Windows / Linux)"
-		buildNumberPattern="test-%build.counter%"
+	name = "Validation of Size Regression - Staging Docker Images (Windows / Linux)"
+	buildNumberPattern="test-%build.counter%"
 
-		vcs {
-			root(TeamCityDockerImagesRepo.TeamCityDockerImagesRepo)
-		}
+	vcs {
+		root(TeamCityDockerImagesRepo.TeamCityDockerImagesRepo)
+	}
 
-		triggers {
-			// Execute the build once the images are available within %deployRepository%
-			finishBuildTrigger {
-				buildType = "${PublishHubVersion.publish_hub_version.id}"
-			}
+	triggers {
+		// Execute the build once the images are available within %deployRepository%
+		finishBuildTrigger {
+			buildType = "${PublishHubVersion.publish_hub_version.id}"
```

Comment:
This is the wrong dependency (at this time images are published to the docker hub, and we want to check it on the staging). So, the right dependency would be the TC_Trunk_DockerImages_publish_local configuration.